command-parser.js

```diff
@@ -7,7 +7,7 @@
  *
  * Individual commands are put in:
  *   commands.js - "core" commands that shouldn't be modified
- *   config/commands.js - other commands that can be safely modified
+ *   chat-plugins/info.js - other commands that can be safely modified
  *
  * The command API is (mostly) documented in config/commands.js
```

Comment:
Putting it in a ```.MD``` file might be a good idea. I should probably also explain how the help command works in it so that people know that the help command has to match the primary function name. Further, all files that include commands could direct people to the single ```.MD``` file which could in turn have a directory for how the different commands are categorized into different files. That might not make much of a difference now, but it would make adding more files less of a headache in the future.